aws_utils/emr/polling_v2.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+import logging
+import time
+from collections import namedtuple
+from datetime import datetime, timedelta
+
+from dateutil.tz import tzlocal
+
+logging.basicConfig()
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.DEBUG)
+
+States = namedtuple('States', ['running', 'success', 'failure'])
+
+CLUSTER_STATES = States(
+    ['STARTING', 'BOOTSTRAPPING', 'RUNNING', 'WAITING', 'TERMINATING'],
+    ['TERMINATED'],
+    ['TERMINATED_WITH_ERRORS'])
+
+
+class EMRPollingException(Exception):
+    pass
+
+def on_cluster_status_change(logger, status, prev_state):
+    """Handle EMR cluster status change."""
+    logger.debug('Cluster status changed: %s -> %s', prev_state, status)
+
+
+def on_cluster_success_status(logger, status, reason):
+    """Handle EMR cluster success status."""
+    logger.debug('Cluster success: %s %s', status, reason)
```

Comment:
I see what you mean... I wonder if this is like that and the `logger` is set to `INFO` only, then they would not get any notification at all about the status of the cluster. Perhaps it's a better option to move this up to `INFO` level, @neebone?